CONTRIBUTING.md

```diff
@@ -184,52 +184,55 @@ Ref: Book: Clean Code by Robert C. Martin (Uncle Bob)
 
 This guide is helpful to compile and install Nmstate from source, for installing stable release or other installation methods, pleaser refer to [Nmstate installation guide](https://nmstate.io/user/install.md)
 
-### Dependencies
-
-Nmstate requires NetworkManager 1.26 or later version installed and started.
-
-In order to support specific capabilities, additional packages are required:
-
-OpenVSwitch support through the NM provider requires NetworkManager-ovs.
-
-Extended ovsdb support requires openvswitch-2.11 and python3-openvswitch2.11 or greater.
-
-Team interface support through the NM provider requires NetworkManager-team
-
 ### Clone the source
-```
+```bash
 git clone https://github.com/nmstate/nmstate.git
 cd nmstate
 ```
 
 #### Install build tool
-```
-sudo dnf install cargo # Fedora
-sudo yum install cargo # RHEL
+```bash
+# Install Cargo using rustup (recommended for development)
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+source $HOME/.cargo/env
```

Comment:
@elomscansio , sorry for confusing you, rust up is specifically for the experienced developer,  you can remove the part above, we can ask the users to do `cargo --version`,  if the system does not have the cargo installed, then install cargo from the package manager of the distribution.